### PR TITLE
Improved Xen tutorial

### PR DESCRIPTION
**Pull request by AlwaysLivid**

### CODE REVIEW

File: content/guides/virtualizing/xen.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ date = "2019-12-7"
...
+Virtual instances of operating systems are perfect for all kinds of testing purposes that need to be done in a safe and isolated environment. Therefore, installing Haiku in a virtual machine is an ideal solution for people who do not want to install it on their physical computers but want to become familiar with it.
 
-In this guide, the Haiku operating system is being run under virtual circumstances using Ubuntu Linux, Xen and Virt-Manager. but you can use any distribution of Linux that supports Xen.
+The guide was tested on an Ubuntu Linux installation, but it should work on any Linux distribution that supports Xen.
 
-This tutorial, for demonstration purposes, uses an Anyboot image. It can be obtained [here](/get-haiku). Both the ISO and Anyboot images are available there. Verify the checksums to make sure the downloaded files are not corrupted.
+This tutorial uses an Anyboot image for demonstration purposes. Both the ISO and the Anyboot image can be obtained [here](/get-haiku). Make sure to verify the checksums in order to make sure that your downloaded files are not corrupted.
```
Comment (humdingerb) [inline content/guides/virtualizing/xen.md:12]: We only have anyboot images to download.

File: content/guides/virtualizing/xen.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ pacman -S ovmf
 Lastly, you will have to enable the `libvirtd` daemon, as it is not enabled by default.
```
Comment (humdingerb) [inline content/guides/virtualizing/xen.md:72]:
```suggestion
Lastly, you will have to enable the `libvirtd` daemon, as it is not enabled by default:
```

File: content/guides/virtualizing/xen.md [hunk 3]
```diff
@@ -91,7 +91,7 @@ yum install xen virt-manager
 Also, make sure to enable the `libvirtd` daemon, which is not enabled by default.
```
Comment (humdingerb) [inline content/guides/virtualizing/xen.md:91]:
```suggestion
Also, make sure to enable the `libvirtd` daemon, which is not enabled by default:
```

File: content/guides/virtualizing/xen.md [hunk 4]
```diff
@@ -137,25 +137,25 @@ First, on the virt-manager window, choose `Edit` > `Connection Details`. The Con
...
+Here, you can customize the network address range. The default settings should be alright, simply click **`Forward`**
 
 ![](/files/guides/virtualizing/xen/ip_range.png)
 
 If you want to enable the IPv6 address range, you can customize it here. Otherwise, just click **`Forward`**
 
 ![](/files/guides/virtualizing/xen/ipv6.png)
 
-Here, you can forward the network packets of the VM to a physical network, or keep the virtual network isolated. If you need Internet connection on the VM, you should forward to a physical network connected to the Internet. When you are done, click **`Finish`**
+Here, you can forward the network packets of the virtual machine to a physical network, or keep the virtual network isolated. If you need an Internet connection on the virtual machine, you should forward to a physical network connected to the Internet. When you are done, click **`Finish`**
```
Comment (humdingerb) [inline content/guides/virtualizing/xen.md:152]:
```suggestion
Here, you can forward the network packets of the virtual machine to a physical network, or keep the virtual network isolated. If you need an internet connection on the virtual machine, you should forward to a physical network connected to the internet. When you are done, click **`Finish`**
```

File: content/guides/virtualizing/xen.md (hunk 1 above)
Comment (AlwaysLivid) [reply content/guides/virtualizing/xen.md:12]: I really can't remember why I wrote that in the first place, then.

**Pull request merged**